packages/scripts/webapp/preset/README.md

```diff
@@ -35,6 +35,7 @@ Your folder hierarchy should follow
 ```json
 {
   "preset": "talend",
+  "angular-legacy": true,
```

Comment:
`angular`instead of `angular-legacy` is enough

Reply:
Changed